### DISCUSSION TITLE
Is there any plan to support the brovi E3372-325 modem?

### DISCUSSION THREAD

**Discussion by ureman2** [Salamek/huawei-lte-api#220] [Ideas]
Is there any plan to support the brovi E3372-325 modem? It is very necessary to read SMS from the modem in order to send it to telegram. I tried different versions of different programs, but it doesn't work. The paths have probably changed slightly. Who can help?

**Comment by Salamek** [comment]
Read https://github.com/Salamek/huawei-lte-api/issues/217

**Comment by ureman2** [comment]
Thanks for the reply. The fact is that this model (E3372-325) is a further development of the E3372-320. Therefore, some of the functions work (token, cookies, status, signal levels and some others), but SMS reading is not given in any way, but it is necessary: (and there is not enough knowledge to do this
